db/migrate/20160131182301_add_download_policy_to_katello_repositories.rb

```diff
@@ -0,0 +1,6 @@
+class AddDownloadPolicyToKatelloRepositories < ActiveRecord::Migration
+  def change
+    add_column :katello_repositories, :download_policy, :string
+    update "UPDATE katello_repositories SET download_policy = 'on_demand' WHERE content_type = 'yum'"
```

Comment:
1) Prevent database specific sequel from being used and letting the ORM handle things properly for us
2) That's how we tend to do things everywhere else unless they can't be accomplished through the ORM